pom.xml

```diff
@@ -0,0 +1,36 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.company</groupId>
+  <artifactId>hello-maven</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+  
+  <properties>
+  	<maven.compler.source>16</maven.compler.source>
+  	<maven.compiler.target>16</maven.compiler.target>
+  </properties>
+  
+  <build>
+  	<plugins>
+  		<plugin>
+  			<groupId>org.apache.maven.plugins</groupId>
+  			<artifactId>maven-compiler-plugin</artifactId>
+  			<version>3.8.1</version> 
+  			<configuration>
+  				<source>16</source>
+  				<target>16</target>
+  			</configuration>
+  		</plugin>
+  	</plugins>
+  </build>
+  
+  <dependencies>
+   	<!-- https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 -->
+	<dependency>
+	    <groupId>org.apache.commons</groupId>
+	    <artifactId>commons-lang3</artifactId>
+	    <version>3.12.0</version>
```

Comment:
What is the scope of this dependency? If it's the default scope, what is it? And why do you use it?

Reply:
Is is compile scope, so the dependency is always appear in classpath.